programs/util.c

```diff
@@ -326,6 +326,29 @@ int UTIL_prepareFileList(const char *dirName, char** bufStart, size_t* pos, char
 
 #endif /* #ifdef _WIN32 */
 
+/* Check if the file is precompressed (.zst, .lz4, .gz, .xz).
+YES => Skip the file (return 0)
+NO => return 1
+*/
+
+int UTIL_isCompressedFile(const char *inputName)
+{
+    return compareExtensions(inputName,g_compressedFileExtensions);
+}
+
+int compareExtensions(const char* infilename, const char* extensionList[])
```

Comment:
minor: coding style :
if this function is exposed outside of its unit, its name should respect the same `PREFIX_` convention,
in this case `UTIL_`.

Alternatively, ask yourself if exposing this function is really needed.
If not, don't expose it, a declare it `static`.

Reply:
Removed the function in the latest commit since it is no longer required.